docs/patching.md

```diff
@@ -25,11 +25,11 @@ Patching is the process where you tell the console
 If your fixtures are RDM equipped, the console can patch itself from the
 rig, see [Patching by RDM](./patching/patching-new-fixtures-or-dimmers.md#patching-by-rdm).
 
-On consoles with a System/Run/Program switch, the switch must be set to
+On older consoles with a System/Run/Program switch, the switch must be set to
```

Comment:
I'm not actually convinced this does anything any more but it does no harm to leave in I guess